bgpd/bgp_route.c

```diff
@@ -4468,7 +4468,8 @@ int bgp_nlri_parse_ip(struct peer *peer, struct attr *attr,
 			if (pnt + BGP_ADDPATH_ID_LEN > lim)
 				return BGP_NLRI_PARSE_ERROR_PACKET_OVERFLOW;
 
-			addpath_id = ntohl(*((uint32_t *)pnt));
+			memcpy(&addpath_id, pnt, 4);
+			addpath_id = ntohl(addpath_id);
```

Comment:
can we backport here a3a850a17df1705211cc908ee69382d60bb4cdff as well? or with another pr?

Reply:
@ton31337 just open another PR, they're free :stuck_out_tongue:

Reply:
![image](https://user-images.githubusercontent.com/3352707/72541611-7acbaf00-388b-11ea-8e2e-28ff9de250bb.png)